docs/getting_started/basic/zero-shot-prediction/zero-shot-detection.ipynb

```diff
@@ -0,0 +1,1378 @@
+{
```

Comment:
<p>How do I know when the delegated job is finished?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thesteve0/v51-gs-draft/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>